experiment/workdir.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, base_dir, keep: bool = False):
     os.makedirs(self.run_logs, exist_ok=True)
     os.makedirs(self._corpus_base, exist_ok=True)
     os.makedirs(self.dills, exist_ok=True)
+    os.makedirs(self.fuzz_target, exist_ok=True)
```

Comment:
what's the difference between fuzz_target and raw_targets/fixed_targets? 

If we want to deprecate/replace the older ones, can you please add a comment?